airflow/contrib/operators/gcp_container_operator.py

```diff
@@ -307,7 +307,7 @@ def _set_env_from_extras(self, extras):
         else:
             # Write service account JSON to secure file for gcloud to reference
             service_key = tempfile.NamedTemporaryFile(delete=False)
-            service_key.write(keyfile_json_str)
+            service_key.write(keyfile_json_str.encode('utf-8'))
```

Comment:
in the base GCP hook class, I see a similar code -decorator  It can not be unified?

Reply:
I think we can open another issue to unify this outside this PR.